### PR TITLE
0.20.1 Adding test for automatic schema propapgating.

### PR DESCRIPTION
**Pull request by antip00**
test regarding issue https://github.com/Altinity/clickhouse-operator/issues/903.

test doesn't check every table engine except MaterializedPostgreSQL and RabbitMQ

test fails for now for live view and window view

### CODE REVIEW

File: tests/e2e/manifests/chi/test-038-1-schema-propagation.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    clusters:
+    - name: simple
+      schemaPolicy:
+        replica: All
+        shard: All
+      layout:
+        shardsCount: 1
+        replicasCount: 1
+    files:
+      graphite_rollup.xml: |
```
Comment (alex-zaitsev) [inline tests/e2e/manifests/chi/test-038-1-schema-propagation.yaml:22]: Don't test Graphite, we do not support it

File: tests/e2e/manifests/chi/test-038-1-schema-propagation.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+    podTemplates:
+      - name: multi-volume
+        spec:
+          containers:
+            - name: clickhouse-pod
+              image: clickhouse/clickhouse-server:22.8.6.71
```
Comment (alex-zaitsev) [inline tests/e2e/manifests/chi/test-038-1-schema-propagation.yaml:73]: Use version from template.
```
  useTemplates:
    - name: clickhouse-version
```

**Review by alex-zaitsev — COMMENTED** [review]
Good test, but it partially overlaps with test_013. 

I suggest to name it test_013_1, but add a few more changes but also test that all distribution policies are respected, including DistributedTablesOnly and None. Use one CHI for that to make it fast, just add extra steps that change schema settings for new shards.

Once the new test is merged I will remove old test_013.

**Review by alex-zaitsev — APPROVED** [review]
